template-bare/package.json

```diff
@@ -3,8 +3,9 @@
   "private": true,
   "version": "0.0.0",
   "scripts": {
-    "dev": "convex dev --until-success && convex dashboard && convex dev",
-    "dash": "convex dashboard",
+    "dev": "convex dev",
+    "predev": "convex dev --until-success && convex dashboard",
```

Comment:
This solves the CTRL-C, right? It's a shame that this is not well known, and comments are not easy in package.json